### ISSUE TITLE
Inconsistent Conditions on Pages

### ISSUE THREAD

**Issue by TRexStark** [umbraco/Umbraco.Forms.Issues#833]
Umbraco Forms 10.0.5

When you add a condition to a Page, the conditions seem to be inconsistent and applied to and from the wrong page.

I did the same test on Groups/Fieldsets and it is not replicated - meaning, it's working consistently on fieldsets but not on Pages.

Reproduction
------------

Create a form with 3 Pages.

In page 1 > Group 1 create a Radio Button as "Hide Next" and 2 values "Yes" and "No"

------------

## **Test 1**
Click the Cog icon to modify the settings of Page 2.

Activate conditions, add set them as:
"Hide" _this fieldset if_ "All"
"Hide Next" - "is" - "Yes"

### Expected result

In the **front-end**:
When you select "Yes" in "Hide Next"

The "Next" button in Page 1 should take you to Page 3

### Actual result

In the **front-end**:
When you select "Yes" in "Hide Next"

The "Next" button takes you to Page 2 and the "Next" button in Page 2 is not visible.

------------

## **Test 2**
Remove the condition from Page 2 and add it to Page 1 instead

Again as:
"Hide" _this fieldset if_ "All"
"Hide Next" - "is" - "Yes"

### Expected result

In the **Back Office**, the condition should work when you apply it to Page 2

In the **front-end**:
When you select "Yes" in "Hide Next"

The "Next" button in Page 1 should take you to Page 3

### Actual result

In the **Back Office**, the condition only works if applied to Page 1 as described above in **Test 1**

In the **front-end**:
When you select "Yes" in "Hide Next"

The "Next" button is hidden entirely and you will not be able to move forward, making the condition and the form unusable.

**Comment by AndyButland** [comment]
I think here you are describing an expected result that isn't the intention of this feature (at least in it's current form).  The idea of conditions on the page level is to allow you to show or hide the submit (or next) button, to only be clickable once the conditions are adhered to.  In other words, only allowing the user to proceed once the data collected so far is complete.

The related issue that this implementation was resolving was #18.

It doesn't allow you to skip out pages though.  This is a feature request already under #38 so may be something we look to implement in the future.  I'll close this given the expected behaviour isn't as described and we already have an issue in the tracker for the feature request.

**Comment by TRexStark** [comment]
@AndyButland, maybe the naming conventions of the conditions need to be updated and made clearer as to what they do.

**Comment by AndyButland** [comment]
Would be happy to take suggestions as it has caused confusion, but this is what we have at the moment, which was the clearest I could come up with at the time and can't see obviously how to improve:
![image](https://user-images.githubusercontent.com/1993459/182020347-c4389dcf-5046-4f2f-8a20-aa8f9356281f.png)

**Comment by TRexStark** [comment]
Thanks @AndyButland

I'd reword it as "Hide Next Button on this page"
Also, the condition on the properties themselves "Form Fields/question" say "Hide this fieldset", it should probably say "Hide this Question if"
